src/com/android/launcher3/PagedView.java

```diff
@@ -516,12 +516,12 @@ public void setOnPageTransitionEndCallback(@Nullable Runnable callback) {
     }
 
     @Override
-    public void scrollTo(int x, int y) {
-        x = Utilities.boundToRange(x,
+    public void scrollTo(int xCoordinate, int yCoordinate) {
+        xCoordinate = Utilities.boundToRange(xCoordinate,
                 mOrientationHandler.getPrimaryValue(mMinScroll, 0), mMaxScroll);
-        y = Utilities.boundToRange(y,
+        yCoordinate = Utilities.boundToRange(yCoordinate,
                 mOrientationHandler.getPrimaryValue(0, mMinScroll), mMaxScroll);
-        super.scrollTo(x, y);
```

Comment:
They are the default args in Launcher3, https://github.com/Goooler/Launcher3/blob/2e7514366cc1ae84f5c59a3d8c4365ee4321b13f/src/com/android/launcher3/PagedView.java#L518-L525